cli/LineReader.cpp

```diff
@@ -171,7 +173,7 @@ std::string LineReader::getNextCommand() {
             case '.':
             case '\\':  //  Fall Through.
               // If the dot or forward slash begins the line, begin a command search.
-              if (scan_position == 0) {
+              if (special_char_location == multiline_buffer.find_first_not_of(" \t\r\n")) {
```

Comment:
I think we should keep `scan_position == 0` as well. Otherwise, it seems that we require some special chars before a command string.

Reply:
Yes it should cover all the possible situations.